image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
I don't know. I simply don't understand why having a directory called refs/tags is confusing. Honestly, I am fine with the refs name too; this was mostly to help clear up @runcom's confusion and it seemed reasonable.

What is your use case for having lots of refs pointing to different images in a single image-layout?

The desired UX for this stuff, I think, is like:

```
rkt image import --name mystuff.com/foo/bar /path/to/oci-image-layout
docker import --name mystuff.com/foo/bar /path/to/oci-image-layout
```

In these cases the name and tags make sense. What is your use case?